src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs

```diff
@@ -640,6 +644,25 @@ where symbol.Equals(symbol.ContainingType.FindImplementationForInterfaceMember(m
             }
         }
 
+        private void GenerateExtensionMethods(INamedTypeSymbol symbol, MetadataItem item)
+        {
+            var extensions = new List<string>();
+            foreach (var e in _extensionMethods)
+            {
+                var reduced = e.ReduceExtensionMethod(symbol);
+                if ((object)reduced != null)
+                {
+                    // update reference
+                    var typeGenericParameters = reduced.ContainingType.IsGenericType ? reduced.ContainingType.Accept(TypeGenericParameterNameVisitor.Instance) : EmptyListOfString;
```

Comment:
seems the generated reference id is not right, but the result(definition) isn't influenced

---
In reply to: [65477693](https://github.com/dotnet/docfx/pull/339#discussion_r65477693) [](ancestors = 65477693)